python/lsst/validate/drp/base.py

```diff
@@ -17,9 +17,22 @@
 # You should have received a copy of the LSST License Statement and
 # the GNU General Public License along with this program.  If not,
 # see <https://www.lsstcorp.org/LegalNotices/>.
+"""lsst.validate.drp's Measurement API that handles JSON persistence.
+"""
 
```

Comment:
This might be a moot point if we decide to split `lsst.validate.drp.base` into an independent Python package.